.pre-commit-config.yaml

```
@@ -1,7 +1,16 @@
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.287
+    hooks:
+      - id: ruff
+        args: [--exit-non-zero-on-fix]
```

Make it clearer that the job is scoped to just `Lib/test/` (this will also help differentiate it from other ruff jobs targeting different parts of the repo, if we add them in the future)

```suggestion
        name: Run ruff on Lib/test/
        args: [--exit-non-zero-on-fix]
```

---

(👍 and edited to capitalise "Ruff")